api/config/routes.rb

```diff
@@ -8,124 +8,119 @@
     end
   end
 
-  concern :api do
-    resources :promotions, only: [:show]
-
-    resources :products do
-      resources :images
-      resources :variants
-      resources :product_properties
-    end
+  namespace :api, defaults: { format: 'json' } do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.